tests/microos/verify_setup.pm

```diff
@@ -59,6 +59,12 @@ sub systemd_tests {
 
     foreach my $unit (@$units) {
         my $name = $unit->{name};
+        # tumbleweed nor microos does not come with mkfs.ext4
+        if ($name eq "create_test_file.service") {
+            if (is_tumbleweed || is_microos) {
+                next;
+            }
+        }
```

Comment:
`is_tumbleweed` covers `is_microos`. But anyway I think a single `if` is better than a nested `if`.  =/

```suggestion
        next if ($name eq "create_test_file.service" && is_tumbleweed);
```

Reply:
Will try this with a VR and report back!

Reply:
https://openqa.opensuse.org/tests/4544698 all good with your suggestion.